etc/get_brick_mapping.py

```diff
@@ -0,0 +1,34 @@
+import etcd
+import json
+import uuid
+
+
+integration_id = "89604c6b-2eff-4221-96b4-e41319240240"
+job_id = str(uuid.uuid4())
+
+payload = {
+    "integration_id": integration_id,
+    "run": "tendrl.gluster.flows.BrickMapping",
+    "parameters": {
+        "Cluster.node_configuration": {
+            "67afa32d-883b-45f3-b331-bc96ecd010ba": None,
```

Comment:
I feel instead of `None` we would need to send actual disk details here. correct?